aip_x2_launch/launch/pointcloud_preprocessor.launch.py

```diff
@@ -33,7 +33,7 @@ def launch_setup(context, *args, **kwargs):
         name="concatenate_data",
         remappings=[
             ("~/input/twist", "/sensing/vehicle_velocity_converter/twist_with_covariance"),
-            ("output", "concatenated/pointcloud"),
+            ("output", "concatenated/pointcloud_raw"),
```

Comment:
`concatenated/pointcloud_raw` sounds very weird. 
The topic name should represent its status, I think. So, it should be something like `concatenated/pointcloud` -> `concatenated/downsampled/pointcloud`.

Reply:
@miursh thank you for your comment. My intention is want to keep using `concatenated/pointcloud` input for perception including centerpoint, ground_segmentation, etc.
Currently we have 2 options:
1. Keep output of pointcloud_preprocessor as `concatenated/pointcloud` and change `concatenated/pointcloud_raw` -> some understandable name. For example `concatenated/before_downsampled/pointcloud`?
2. Change output of `pointcloud_preprocessor` and all input `perception` module to `concatenated/downsampled/pointcloud`
I currently prefer to 1) but of course I can change as 2) as your opinion.

Reply:
I prefer another option.
Since I don't want to change the interface between sensing and perception, the output of sensing module should be `concatenated/pointcloud`. 
And if you want to apply donwsampling before perception, you should add a donwsampling node in `/perception` namespace.
BTW, if you change the input point of centerpoint, I want to know that this change does not have much effects on the perception result.

Reply:
@miursh Thank you for your comment, Since I am going to add the downsampling before `common_ground_filter` in `perception` (https://github.com/tier4/autoware.universe/pull/870/commits/bc48f9ef5a1c41dd6ff41c3054d640cb698398f3) , it will not affect on centerpoint input and this PR is unnecessary. I will close this.